ee/tasks/webhooks_ee.py

```diff
@@ -12,41 +12,53 @@
 
 @app.task(ignore_result=True, bind=True, max_retries=3)
 def post_event_to_webhook_ee(self: Task, event: Dict[str, Any], team_id: int, site_url: str) -> None:
-    team = Team.objects.get(pk=team_id)
-    _event = Event.objects.create(
-        event=event["event"],
-        distinct_id=event["distinct_id"],
-        properties=event["properties"],
-        team=team,
-        site_url=site_url,
-        **({"timestamp": event["timestamp"]} if event["timestamp"] else {}),
-        **({"elements": event["elements_list"]} if event["elements_list"] else {})
-    )
-
-    actions = cast(Sequence[Action], Action.objects.filter(team_id=team_id, post_to_slack=True).all())
-
     if not site_url:
         site_url = settings.SITE_URL
+    postgres_event = None
+    try:
+        team = Team.objects.select_related("organization").get(pk=team_id)
+        postgres_event = Event.objects.create(
```

Comment:
Huh, this is quite sus. 

I see this was in the previous version as well, but perhaps we should:
1. Explicitly wrap this code into a transaction
2. Roll back at the end

This way we can make sure no temporary events stay polluting the database.

Reply:
I don't think we really care about 100% cleanliness in this case and transaction incur some performance penalty, so I'd rather leave it. The `finally` should be handling this well in all but very exceptional cases.

Reply:
I was more worried about data hygene here rather than code cleanliness - celery tasks may be aborted in the middle in which case we'd always end up with an extra event unless we wrap this in a transaction (which would get automatically rolled back). 

Performance-wise since we skip the need to do a 'real' insert I imagine it's comparable?

Reply:
Yes, I meant data hygiene too, but anyway alright, switched to a transactional approach